Camera2D.gd

```diff
@@ -0,0 +1,16 @@
+extends Camera2D
+
+
+const DEAD_ZONE: int = 140
+@onready var alpaca = $"../Alpaca"
```

Comment:
Inflexible. What if we want the target to not to be the Alpaca?

You can make it more general with
@export var target_scene: PackedScene 
and in the editor drag and drop the scene that you want as target.